common/src/test/java/tachyon/util/io/FileUtilsTest.java

```diff
@@ -99,4 +115,53 @@ public void deleteNonExistentFileTest() throws IOException {
     mException.expect(IOException.class);
     FileUtils.delete(ghostFile.getAbsolutePath());
   }
+
+  @Test
+  public void setLocalDirStickyBitTest() throws IOException {
+    File tempFolder = mTestFolder.newFolder("dirToModify");
+    // Only test this functionality of the absolute path of the temporary directory starts with "/",
+    // which implies the host should support "chmod".
+    if (tempFolder.getAbsolutePath().startsWith(TachyonURI.SEPARATOR)) {
+      FileUtils.setLocalDirStickyBit(tempFolder.getAbsolutePath());
+      List<String> commands = new ArrayList<String>();
+      commands.add("/bin/ls");
+      commands.add("-ld");
+      commands.add(tempFolder.getAbsolutePath());
+      try {
+        ProcessBuilder builder = new ProcessBuilder(commands);
+        Process process = builder.start();
+        process.waitFor();
+        BufferedReader stdInput = new BufferedReader(new
+            InputStreamReader(process.getInputStream()));
+        String line = stdInput.readLine();
+        Assert.assertTrue(line.startsWith("drwxr-xr-t"));
+      } catch (InterruptedException e) {
+        e.printStackTrace();
+      }
+    }
+  }
+
+  @Test
+  public void createBlockPath() throws IOException {
```

Comment:
`createBlockPath` -> `createBlockPathTest` ?
Also for the other few tests below